app/scripts/modules/core/src/pipeline/config/stages/gremlin/gremlinStage.ts

```diff
@@ -0,0 +1,19 @@
+import { Registry } from 'core/registry';
+
+import { ExecutionDetailsTasks } from '../core';
+import { GremlinExecutionDetails } from './GremlinExecutionDetails';
+import { GremlinStageConfig } from './GremlinStageConfig';
+
+Registry.pipeline.registerStage({
```

Comment:
We need to put this registration behind a feature flag.
```suggestion
import { SETTINGS } from 'core/config/settings';

if (SETTINGS.feature.gremlin) {
  Registry.pipeline.registerStage({
```

Reply:
then add the feature flag in settings.ts and the various settings.js files.